### PR TITLE
add InfoBox

### PR DESCRIPTION
**Pull request by nickhsine**
1. add InfoBox
2. remove /* eslint no-unused-vars:1 */ 
3. Update Image.js. Replace local css by global css

### CODE REVIEW

**Comment by hcchien** [comment]
lgtm

File: src/themes/bootstrap.overrides.scss [hunk 1]
```diff
@@ -15,6 +15,12 @@
   }
 }
 
+.image-caption {
+  font-size: 14px;
+  color: #242423;
```
Comment (garfieldduck) [inline src/themes/bootstrap.overrides.scss]: define the color in common-variables (?

File: src/themes/bootstrap.overrides.scss [hunk 2]
```diff
@@ -15,6 +15,12 @@
   }
 }
 
+.image-caption {
+  font-size: 14px;
```
Comment (garfieldduck) [inline src/themes/bootstrap.overrides.scss]: you may use $font-size-small

Comment (nickhsine) [inline src/themes/bootstrap.overrides.scss]: use it in the latest commit

**Comment by garfieldduck** [comment]
👍 👍 👍

**Comment by nickhsine** [comment]
https://github.com/twreporter/twreporter-react/issues/189

**Pull request merged**